src/pages/backstop.tsx

```diff
@@ -126,17 +124,16 @@ const Backstop: NextPage = () => {
         setAvailableToMint('0');
       }
     } catch (e) {
-      console.error('error on claim fn ');
+      console.error('Unable to estimate LP token mint amounts');
       setLoadingEstimate(false);
       setAvailableToMint('0');
     }
   }
 
   useEffect(() => {
-    if (balancesByAddress.get(backstopData?.config.usdcTkn ?? '') === undefined) {
-      loadUserData(walletAddress);
```

Comment:
had added this to load usdc and blnd balances in case they're not there on page load

Reply:
they should be, and this caused a situation where if you were on this page and the request failed for any reason, (like a rate limit), we begun spamming requests to the endpoint, causing long blocking times.

On normal usage, this was almost always loaded for me, so I think we should remove the potential for looping requests, and address this a different way if we find a way where it is still an issue.